contracts/interchain-token-service/src/lib.rs

```diff
@@ -2,15 +2,21 @@
 
 #[cfg(any(test, feature = "testutils"))]
 extern crate std;
-mod abi;
-mod contract;
+
 pub mod error;
 pub mod event;
+pub mod executable;
 mod interface;
-mod storage_types;
-mod token_handler;
 pub mod types;
 
-pub mod executable;
-
-pub use contract::{InterchainTokenService, InterchainTokenServiceClient};
+cfg_if::cfg_if! {
+    if #[cfg(all(feature = "library", not(feature = "testutils")))] {
+        pub use interface::{InterchainTokenServiceClient, InterchainTokenServiceInterface};
+    } else {
+        mod abi;
```

Comment:
Is there any reason not to include the event?
```suggestion
        mod abi;
        mod event;
```